### PR TITLE
UI: Dashboard GitHub badges

### LINKED ISSUE #172: UI: Dashboard GitHub badges
We should provide a button with a pop-up modal dialog that lets the user copy/paste markdown for a GitHub badge. This initial version won't connect to a live service but could just say `[OpenFaaS: Cloud]` for instance. Should be on the details page. See Travis CI for an example of this or the OpenFaaS readme.

### PR DESCRIPTION
**Pull request by jaigouk**
Signed-off-by: Jaigouk Kim <ping@jaigouk.kim>

## Description

- fixes #172
- added react-bootstrap
- added a button to open a modal that has pre-filled text input

## How Has This Been Tested?

manually tested

get-badge

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [x] read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

**Comment by alexellis** [comment]
Thank you very much for this PR. Can I ask you to add a quick gif showing us how this looks?

**Comment by alexellis** [comment]
I've just seen the gif, thanks for that. I think we do need a copy icon on the modal as well.

Once that is in place if Ken, Jag or Bart give LGTM then I will go ahead of merge. Feel free to ping them in the public channel on Slack 

Alex

**Comment by kenfdev** [comment]
Very nice @jaigouk ! Is it possible to show a preview of the badge?

**Comment by jaigouk** [comment]
something like this?

screen shot 2018-09-21 at 02 07 22

**Comment by kenfdev** [comment]
Nice!

> I think we do need a copy icon on the modal as well.

As Alex mentioned, it'd be good experience to have a copy icon

**Comment by jaigouk** [comment]
copy-badge-md
umm... sorry i was too late for this.
@alexellis

**Comment by alexellis** [comment]
Let's add a very brief message in plain English saying what to do with it.. i.e. about adding it to your GitHub README

**Comment by alexellis** [comment]
Thanks for this PR!

**Comment by jaigouk** [comment]
@alexellis 
here is the change https://github.com/openfaas/openfaas-cloud/compare/master...jaigouk:copy-badge-markdown?expand=1

I will create a PR today with the brief message.
